### ISSUE TITLE
I want the ability to create multiple colour presets that I can apply to charts

### ISSUE THREAD

**Issue by PriPatel** [lightdash/lightdash#4364]
> It would be nice to have many groups of presets. Like 5 colors per named preset, and in the appearance section of the chart, you can select from the presets.
> From there you can click the individual grouped metric to choose which metric gets which preset.
> I spend a lot of time typing in hex codes.
> 

- Go to the `Appearance` tab under Settings
- Hit `Create new preset`
- Name your preset
- Maybe eventually you can pick what type of preset it is (eg, perhaps we can easily calculate a `divergent` palette for the user if they specify 2 hexcodes)
- Once you have entered the hex codes, hit `Save`.
- When you're in chart config mode, you should be able to select which metric gets which preset
- You should be able to edit and delete your presets  

Related to https://github.com/lightdash/lightdash/issues/1579
Blocked by https://github.com/lightdash/lightdash/issues/1689

**Comment by the-watchmaker** [comment]
It's better to keep the colour preset setting near the tables. 
I've noticed that many users use it as a presentation chart generator. 
Going back and forth from the appearance to the working chart can be cumbersome.
